src/GatewayConnectionManager.cpp

```diff
@@ -207,6 +207,15 @@ bool ConnectToLCG() {
 
   _lastConnectionAttempt = msNow;
 
+  if (Config::HasBackendLCGOverride()) {
+    std::string lcgOverride;
+    Config::GetBackendLCGOverride(lcgOverride);
+
+    ESP_LOGD(TAG, "Connecting to overridden LCG endpoint %.*s", lcgOverride.size(), lcgOverride.data());
+    s_wsClient->connect(lcgOverride.c_str());
+    return true;
+  }
+
```

Comment:
I also noticed the getting-backend segment can return `true` before we verify we have an Auth token to give to the gateway. Is that intended?

Reply:
This is outside the scope of this PR, but is fixed in a seperate commit